migrations/versions/0345_move_broadcast_provider.py

```diff
@@ -1,16 +1,16 @@
 """
 
 Revision ID: 0344_move_broadcast_provider
```

Comment:
This Revision ID should be 0345_move_broadcast_provider

Reply:
Just fixed this and rebased